com.nomachine.nxplayer.desktop

```
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Encoding=UTF-8
+Comment=Access NoMachine computers over the network
+Exec="/app/extra/NX/bin/nxplayer"
+Icon=com.nomachine.nxplayer.png
```

it the icon **name**. 
```suggestion
Icon=com.nomachine.nxplayer
```